### DISCUSSION TITLE
can't log into windows app

### DISCUSSION THREAD

**Discussion by tomas02061991** [ente-io/ente#2776] [Q&A]
When i try putting in my data to create an account i got "something went wrong" i then created an account thruogh the web, but when trying to login to said account on windows app, i get the same error: "something went wrong"
![Capture](https://github.com/user-attachments/assets/ace8f5bb-9ef9-43f0-b4d0-3b7d379ce2d6)

**Comment by acleiker2** [comment]
I am having the same issue.
